javascript/app.js

```diff
@@ -2,15 +2,215 @@ const menuBars = document.getElementById('menu-bars');
 const mobileMenu = document.getElementById('mobile-menu');
 const closeMenuBtn = document.getElementById('close-menu-btn');
 const mobileLinks = document.getElementById('mobile-links');
+const workModal = document.getElementById('work-modal');
 
-function openMenuHandler() {
+const preventDefaultHandler = (e) => {
+  e.preventDefault();
+};
+
+const openMenuHandler = () => {
   mobileMenu.classList.replace('d-none', 'd-flex');
-}
+  document.body.classList.add('stop-scrolling');
+  document.body.addEventListener('touchmove', preventDefaultHandler);
+};
 
 function closeMenuHandler() {
   mobileMenu.classList.replace('d-flex', 'd-none');
+  document.body.classList.remove('stop-scrolling');
+  document.body.removeEventListener('touchmove', preventDefaultHandler);
 }
 
 menuBars.addEventListener('click', openMenuHandler);
 closeMenuBtn.addEventListener('click', closeMenuHandler);
-mobileLinks.addEventListener('click', closeMenuHandler);
\ No newline at end of file
+mobileLinks.addEventListener('click', closeMenuHandler);
+
+const worksArray = [
+  {
+    id: 0,
+    title: 'Tonic',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: '/images/work_1.jpg',
+    urlDesktop: '/images/work_1_lg.png',
+    urlModal: '/images/work_modal.png',
```

Comment:
- [ ] [OPTIONAL] The images didn't work in the reviewer's machine because you need to start the path of the image with a dot like this `urlMobile: './images/work_1.jpg',`. For the next milestone take care of that to solve this conflict.